webrtc.html

```diff
@@ -625,6 +625,313 @@
+                <var>description</var>'s <code><a href=
+                "#widl-RTCSessionDescription-type">type</a></code> is wrong for
+                the current <a href=
+                "#dom-peerconnection-signaling-state">signaling state</a> of
+                <var>connection</var>, then reject <var>p</var> with an
+                <code>InvalidSessionDescriptionError</code> and abort these
+                steps.</p>
+              </li>
+
+              <li>
+                <p>If <var>description</var> cannot be applied at the media
+                layer, but the User Agent recovered, possibly by rolling back
+                to the previous configuration, then reject <var>p</var> with
+                <code>IncompatibleSessionDescriptionError</code> and abort
+                these steps.</p>
+              </li>
+
+              <li>
+                <p>Set <var>connection</var>'s <a href=
+                "#dom-peerconnection-signaling-state">signaling state</a> to
+                <code>closed</code>.</p>
+              </li>
+
+              <li>
+                <p>Fire a simple event named <code><a href=
+                "#event-signalingstatechange">signalingstatechange</a></code>
+                at <var>connection</var>.</p>
+              </li>
+
+              <li>
+                <p>Reject <var>p</var> with <code>InternalError</code>.</p>
+              </li>
+            </ol>
+          </li>
+
+          <li>
+            <p>If <var>description</var> is applied successfully, the user
+            agent must queue a task that runs the following steps:</p>
+
+            <ol>
+              <li>
+                <p>If <var>connection</var>'s <a href=
+                "#dom-peerconnection-signaling-state">signaling state</a>
+                is <code>closed</code>, then abort these steps.</p>
+              </li>
+
+              <li>
+                <p>If <var>description</var> is set as a local description, and
+                its content matches the state of all tracks and data channels,
+                <a href="#clearing-negotiation-needed">as defined below</a>,
+                clear the negotiation-needed flag.
+                </p>
+              </li>
+
+              <li>
+                <p>Set <var>connection</var>'s description attributes
+                and <a href=
+                "#dom-peerconnection-signaling-state">signaling state</a> by
+                executing one of the following:</p>
+
+                <p class="issue">NOTE: The principles of pending and current SDP
+                were agreed by the WG but the details the this step has not
+                yet been fully reviewed. TODO - review this.</p>
+
+                <ul>
+                  <!-- A) transition stable to haveLocalOffer -->
+                  <li>
+                    <p>If <var>description</var> is set as a local description
+                    of type "offer", and it has a version that is later than
```

Comment:
The version isn't relevant here.  Checking the version is a precondition on applying the description, not a post-condition.